addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -56,6 +56,7 @@ import {
     isVoidElement,
     cleanZWS,
     isZWS,
+    getDeepestPosition,
```

Comment:
About the commit message:
- would appear => would appear
- viually => visually
- "how many person" sounds weird to me, I would have used "how many people" but I don't know for sure, it might have been valid all along
- retrieve => retrieve

Reply:
Commit message changed.